src/hooks/usePageLocation.ts

```diff
@@ -0,0 +1,16 @@
+import { useLocation } from 'react-router-dom'
+
+import { APP_PATHS } from 'constants/index'
+
+const SWAP_PAGE_URLS = [APP_PATHS.SWAP, APP_PATHS.PARTNER_SWAP, APP_PATHS.LIMIT, APP_PATHS.CROSS_CHAIN]
+
+const usePageLocation = () => {
```

Comment:
when url /crosschain
isSwapPage:true, isCrossChain:true 
is this correct?

Reply:
Yes, It's as expected